### ISSUE TITLE
Suggestion: Ask for storage permission after updating

### ISSUE THREAD

**Issue by Wander29** [hardcore-sushi/DroidFS#229]
First of all, thank you for this awesome app, I'm happy to use it and I share with friends ( also non programmers ;) )
I'd like to inform you that I had a moment of panic when I updated the app and I tried to open my usual vault (update from 1.10.x to 2.0.2, fdroid). I had this error message after typing the correct password:

**The configuration file cannot be loaded. Make sure the volume is accessible**
The corresponding string should be `config_load_error` 

I solved it trying to create a new vault, because when clicking on the folder icon it showed me the request for storage support. I suppose that it must be renabled after an update (at least with this big update 1.x to 2.x?).

If this is the case, it is better to ask it at the very start of the app after the update in my opinion
Thank you

**Comment by hardcore-sushi** [comment]
Yes, it's a known bug, but I don't think it's worth implementing a fix now, as there's less chance of updating from 1.* to 2.0.3 than from 2.0.* to 2.0.3. Maybe a troubleshooting section in the README could be enough.

**Comment by Wander29** [comment]
Thank you for answering. I didn't find a relative issue, I'm sorry if this is a duplicate. Since you know about this you can close if you like.

I agree with you that few months have gone since the update jump and that there's less chance, yet I think it is worth doing this fix for completeness, since it is relatively easy (check version at start and show dialog to grant permission if you are in this case), other than adding it to the README. Just my two cents.

**Comment by hardcore-sushi** [comment]
Yes it's mixed with another issue: #197

I'm too lazy to implement this right now so any PR is welcome.
